packages/runtime/test/dataViews/requestItems/CreateRelationshipAttributeRequestItemDVO.test.ts

```diff
@@ -87,9 +87,10 @@ beforeAll(async () => {
 afterAll(() => serviceProvider.stop());
 
 beforeEach(async () => {
-    rEventBus.reset();
-    sEventBus.reset();
     await cleanupAttributes([sRuntimeServices, rRuntimeServices]);
+
+    sEventBus.reset();
+    rEventBus.reset();
```

Comment:
I don't think you have to move that call. The only thing that's reset is the mocking stuff and not the actual eventbus.. or did it break when it was called after?